Test-network.md

```diff
@@ -140,23 +140,59 @@ $ storj-sim network run
+By default, storj-sim will start a redis server for each Satellite that is run and write the respective connection string for each database to a redis.conf file. You must have `redis-server` installed locally for this process to run. You can download redis at https://redis.io/download. (On macOS you can install redis with homebrew `brew install redis`)
+
+For example, if you're running 2 satellites...
+
+```
+storj-sim network --satellites 2 setup
+storj-sim network --satellites 2 run
+```
+
+...a configuration like the following may be generated.
+
 ```
 // satellite 0
---live-accounting.db: redis://127.0.0.1:5678?db=0
---server.revocation-dburl: redis://127.0.0.1:5678?db=1
+--live-accounting.db: redis://127.0.0.1:10004?db=0
+--server.revocation-dburl: redis://127.0.0.1:10004?db=1
 ```
 
 ```
 // satellite 1
---live-accounting.db: redis://127.0.0.1:5678?db=2
---server.revocation-dburl: redis://127.0.0.1:5678?db=3
+--live-accounting.db: redis://127.0.0.1:10014?db=0
+--server.revocation-dburl: redis://127.0.0.1:10014?db=1
+```
+
+Alternatively, you can set up your own redis server and connect storj-sim directly. 
+
+For example, you may run `redis-server` (default port = `6379`) or `redis-server --port <port>` then `storj-sim network setup --redis 127.0.0.1:<port>`. You can also set an environment variable for $STORJ_SIM_REDIS, e.g. `export STORJ_SIM_REDIS=127.0.0.1:<port>` then simply run `storj-sim network setup` without passing in a flag. All Satellites will use the same server but different dbs if you run a redis server yourself.
+
+Example for 2 satellites:
+```
+storj-sim network --satellites 2 --redis 127.0.0.1:6379 setup
+storj-sim network --satellites 2 run
+```
+OR
+```
+export STORJ_SIM_REDIS=127.0.0.1:6379
+storj-sim network --satellites 2 setup
+storj-sim network --satellites 2 run
+```
+results in:
+
+```
+// satellite 0
+--live-accounting.db: redis://127.0.0.1:6379?db=0
+--server.revocation-dburl: redis://127.0.0.1:6379?db=1
 ```
 
-However, on setup, you may pass in a redis address for a `redis-server` you're already running locally like in this example: `storj-sim network setup --redis redis://127.0.0.1:6379` 
+```
+// satellite 1
+--live-accounting.db: redis://127.0.0.1:6379?db=2
+--server.revocation-dburl: redis://127.0.0.1:6379?db=3
+```
 
-Note: 
-To set up a redis server locally, you can download redis here: https://redis.io/download. Once downloaded, simply run `redis-server` or `redis-server -port <port>` in a separate terminal window.
+If you're running your own redis server, you can designate the database value to start with flag `--db-start` which defaults to `0`. For example, `storj-sim network --satellites 2 --redis 127.0.0.1:6379 --db-start 3 setup` will start incrementing the dbs from 3 (db=3,4,5,6 for 2 satellites). It assumes all databases from the start value onward are available.
```

Comment:
the only thing that would be nice to change (i think egon commented on this in your PR) is to name `--db-start` something like `--redisdb-start` so its clear its only for redis and not all dbs